apicast/conf/nginx.conf

```diff
@@ -4,6 +4,7 @@ env REDIS_URL;
 env RESOLVER;
 env BACKEND_ENDPOINT_OVERRIDE;
 env RHSSO_ENDPOINT;
+env APICAST_OAUTH_ACCESS_TOKEN_TTL;
```

Comment:
All variables starting with `APICAST_` are forwarded automatically. This is not necessary.